github/compliance/issue.py

```diff
@@ -85,23 +90,13 @@ def repository_labels(
 
 @github.retry.retry_and_throttle
 def enumerate_issues(
-    component: cm.Component | None,
-    artifact: cm.Artifact | None,
+    labels: frozenset[str],
```

Comment:
hmm, actually I do not think we need a frozenset - any iterable should do

Reply:
We at least have to use collections, see line 103